main/export-list.js

```diff
@@ -261,4 +262,27 @@ const callExportsWindow = (channel, data) => {
 
 module.exports = () => {
   exportList = new ExportList();
+
+  app.on('before-quit', event => {
+    if (exportList.currentExport) {
+      openExportsWindow();
+      const exportsWindow = getExportsWindow();
+
+      const buttonIndex = dialog.showMessageBoxSync(exportsWindow, {
+        type: 'question',
+        buttons: [
+          'Quit',
+          'Continue'
+        ],
+        defaultId: 1,
+        cancelId: 0,
+        message: 'Do you want to continue exporting?',
+        detail: 'Kap is currently exporting files. If you quit, the export task will be cancelled.'
```

Comment:
Do we go for US English? If so, to be consistent: 
```suggestion
        detail: 'Kap is currently exporting files. If you quit, the export task will be canceled.'
```

Reply:
I think `cancelled` is more common in the US too. Even macOS uses `cancelled` for US English, see #715.